fft_folder/test_preparation.py

```diff
@@ -1,72 +1,37 @@
 """
 test_preparation.py
```

Comment:
docstring should maybe include a small bit about what tests are included or what functionalities are being tested for posterity.